src/gwt/src/org/rstudio/core/client/widget/LeftRightToggleButton.java

```diff
@@ -85,10 +86,21 @@ public LeftRightToggleButton(String leftLabel, String rightLabel,
       else
          addStyleName(styles.rightOn());
 
+      setClassId();
       Roles.getButtonRole().setAriaPressedState(left_, leftIsOn ? PressedValue.TRUE : PressedValue.FALSE);
       Roles.getButtonRole().setAriaPressedState(right_, leftIsOn ? PressedValue.FALSE : PressedValue.TRUE);
    }
 
+   public void setClassId()
+   {
+      ClassIds.assignClassId(getElement(),
+         ClassIds.LEFT_RIGHT_TOGGLE_BTN + " _ " +
+            left_.getInnerText() + "_" +
+            right_.getInnerText());
+      ClassIds.assignClassId(left_, ClassIds.LEFT_TOGGLE_BTN + "_"  + left_.getInnerText());
```

Comment:
Same as other question, does this potentially result in class ids with spaces in them?

Reply:
You're right, thanks! I've fixed the issue.